src/System/Process/PID1.hs

```diff
@@ -1,12 +1,21 @@
 {-# LANGUAGE CPP #-}
 module System.Process.PID1
-    ( run
+    ( RunOptions
+    , defaultRunOptions
+    , run
+    , runEnv
```

Comment:
Sure, makes sense. Is it ok to expose getters as well, e.g. `getRunEnv :: RunOptions -> [(String, String)]`? Otherwise, I don't see how I can implement the `options` method in `Main.hs`, which allows for multiple `-e` flags to build up the environment array